flaml/automl.py

```diff
@@ -1402,44 +1565,44 @@ def fit(
         y_train=None,
         dataframe=None,
         label=None,
-        metric="auto",
-        task="classification",
-        n_jobs=-1,
+        metric=None,
+        task=None,
+        n_jobs=None,
         gpu_per_trial=0,
-        log_file_name="flaml.log",
-        estimator_list="auto",
-        time_budget=60,
-        max_iter=1000000,
-        sample=True,
-        ensemble=False,
-        eval_method="auto",
-        log_type="better",
-        model_history=False,
-        split_ratio=SPLIT_RATIO,
-        n_splits=N_SPLITS,
-        log_training_metric=False,
-        mem_thres=MEM_THRES,
-        pred_time_limit=np.inf,
-        train_time_limit=np.inf,
+        log_file_name=None,
+        estimator_list=None,
+        time_budget=None,
+        max_iter=None,
+        sample=None,
+        ensemble=None,
+        eval_method=None,
+        log_type=None,
+        save_best_model_per_estimator=None,
```

Comment:
Is it necessary to change 'model_history' to `save_best_model_per_estimator`? Won't be compatible with code using older versions.

Reply:
I can rename back.